pkg/skaffold/runner/deploy.go

```diff
@@ -126,6 +126,15 @@ func (r *SkaffoldRunner) loadImagesIntoCluster(ctx context.Context, out io.Write
 		}
 	}
 
+	if config.IsMicrok8sCluster(r.runCtx.GetKubeContext()) {
+		microk8sCluster := config.Microk8sClusterName(currentContext.Cluster)
```

Comment:
`config.IsMicrok8sCluster` function can be moved to `deploy` package and defined as a local  function as it does not rely on any utilities in `config`